docs/locale/fr/LC_MESSAGES/userguide/installation.po

```diff
@@ -192,14 +192,13 @@ msgid "Finally, reboot your Mac to complete the installation."
 msgstr "Enfin, redémarrez votre Mac pour finaliser l'installation."
 
 #: ../../userguide/installation.rst:117
-#, fuzzy
 msgid ""
 "On macOS 12 and older, if the installer fails to start, check the `App Store "
```

Comment:
This was a typo, the recent change to the french version was correct, I should have changed the english version as well. It's on versions >=12. This reminds me that this guide should be updated, the screenshots are obsolete and parts of the instructions as well.

Reply:
OK, so I will close this PR and create an issue for fixing this translation and updating the instructions.

Reply:
#10172